### PR TITLE
feat(icons): add icon of pix colerful

### PR DESCRIPTION
**Pull request by Natalie9**
## Infos

[Task](https://juntossomosmais.monday.com/boards/3345740155/views/76424285/pulses/7072708817)

#### What is being delivered?

Add new color photo icon

#### What impacts?

Icon package 

#### Reversal plan

Follow documentation an revert this PR
#### Evidences

Media(images, gifs or videos) that shows the result of your work.

### CODE REVIEW

**Review by mauriciomutte — APPROVED** [review]

**Review by tiago-dela-rosa — DISMISSED** [review]

File: packages/core/src/icons.d.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export type IconProps =
   | 'leader-board'
   | 'person-time-outline'
   | 'person-warning-outline'
+  | 'pix-colorful'
```
Comment (felipefialho) [inline packages/core/src/icons.d.ts:12]: We need to add this info in the i[con documentation](https://juntossomosmais.github.io/atomium/?path=/docs/components-icon--docs) to explain out approach about colorful icons

**Review by felipefialho — CHANGES_REQUESTED** [review]

File: packages/core/src/icons.d.ts (hunk 1 above)
Comment (Natalie9) [reply packages/core/src/icons.d.ts:12]: My suggestion 👇 
What do you think? 

image

Comment (felipefialho) [reply packages/core/src/icons.d.ts:12]: Perfect, we could add `` around `-colorful` to improve legibility

Comment (Natalie9) [reply packages/core/src/icons.d.ts:12]: https://github.com/juntossomosmais/atomium/pull/525/commits/c449af971038745261623db64f9db2fb098774eb

**Review by felipefialho — APPROVED** [review]

**Review by MayaraRMA — APPROVED** [review]

**Review by tiago-dela-rosa — APPROVED** [review]

**Pull request merged**